ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -528,10 +527,11 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
           id = "LEDGER_ID_MISMATCH",
           ErrorCategory.InvalidGivenCurrentSystemStateResourceMissing,
         ) {
-      case class Reject(override val cause: String)(implicit
+      case class Reject(_expectedLedgerId: String, _receivedLegerId: String)(implicit
```

Comment:
not related to your PR, but I am wondering from where are these underscores coming from?

Reply:
These errors get converted into `StatusRuntimeException`s and we have an introspection based piece of code that gathers all not-underscored fields into details of the said `StatusRuntimeException`s. 
So using underscores here to prevent that from happening.

Reply:
ah i see, thanks for the explanation ;)